usecases/MOC_integration/moc_integration.adoc

```diff
@@ -2,80 +2,82 @@
 
 == Abstract
 
+The content of this repo is under Openshift 3.x, now it is under the process to be updated to version 4.x.
+
 This document describes the steps that are required to get ChRIS with `pfcon` and `pman` up and running on the MOC. The intended audience is ChRIS developers.
 
 . xref:#registering-to-mass-open-cloud[Registering to Mass Open Cloud]
-. xref:#adding-projects-to-openshift-and-openstack[Adding Projects to Openshift and Openstack]
+. xref:#adding-projects-to-openshift[Adding Projects to Openshift]
```

Comment:
ASCIIDoc has a table of contents feature.

https://docs.asciidoctor.org/asciidoc/latest/syntax-quick-reference/#automatic-toc